controllers/notifyController.php

```diff
@@ -226,8 +259,9 @@ private function checkMerchantOrderStatus()
                           " Merchant order id:".$this->woocommerceOrder->get_id().
                           " Merchant order status:".$this->woocommerceOrder->get_status().
                           " Pagantis order id:".$this->pagantisOrder->getStatus().
-                          " Pagantis order status:".$this->pagantisOrder->getId();
-
+                          " Pagantis order status:".$this->pagantisOrder->getId().
+                          " Pagantis urlToken: ".$this->getVerificationToken();
```

Comment:
It is not a revelant info for the mechant

Reply:
modified